### PR TITLE
Remove shared instance

### PR DESCRIPTION
**Pull request by Tedc0819**

### CODE REVIEW

File: lib/services/Logger.js [hunk 1]
```diff
@@ -63,8 +51,7 @@ class Logger {
       return
     }
 
-    handler(this, ...args)
-  }
+    handler(this, ...args) }
```
Comment (jas-chen) [inline lib/services/Logger.js]: why change this?

Comment (Tedc0819) [reply lib/services/Logger.js]: typo. will fix this

File: lib/services/MessageQueue.js [hunk 2]
```diff
@@ -4,24 +4,13 @@
...
-    if (!_messageQueueLib) {
-
-      _messageQueueLib = require('amqplib')
-
-    }
-
-    return _messageQueueLib
-
+  constructor(config) {
+    this.init(config)
```
Comment (jas-chen) [inline lib/services/MessageQueue.js]: just rename `init` to `constructor`?

Comment (Tedc0819) [reply lib/services/MessageQueue.js]: there are some cases that we need to separate the function. I can't deny that this case seems a bit redundant. Let refactor all the constructor and init function. after these PRs.

File: test/integration/logger-event-callback/didLog.test.js [hunk 3]
```diff
@@ -71,7 +71,5 @@ class TestSuite extends TestCombo {
 
   failureAssert(test, combination) {
   }
-}
-
-const testSuite = new TestSuite()
+} const testSuite = new TestSuite()
```
Comment (jas-chen) [inline test/integration/logger-event-callback/didLog.test.js]:
```suggestion
}

const testSuite = new TestSuite()
```

File: lib/services/MessageQueue.js (hunk 2 above)
Comment (jas-chen) [reply lib/services/MessageQueue.js]: it's weird that an object can init more than once 🤔

**Pull request merged**